JetMETCorrections/Modules/interface/CorrectedJetProducer.h

```diff
@@ -60,15 +57,17 @@ namespace reco {
   }
 
   template<class T>
-  void CorrectedJetProducer<T>::produce(edm::Event& fEvent,
-					 const edm::EventSetup& fSetup)
+  void CorrectedJetProducer<T>::produce(edm::StreamID, edm::Event& fEvent, const edm::EventSetup& fSetup) const
   {
+    // cache - note that variable length arrays are a GCC extension
+    reco::JetCorrector const * correctors[mCorrectorTokens.size()];
```

Comment:
I'm not against avoiding the extensions, but if we do want to enforce it, can we `-Werror=vla` to compiler options to prevent it ?